airbnb-for-work.html

```diff
@@ -1020,7 +1020,7 @@ <h3 class="footer__heading">Airbnb</h3>
                           </section>
                         </div>
                         <div class="footer__copyright">
-                          <span class="a8jt5op dir dir-ltr"
+                          <span class="text__highlight dir dir-ltr"
```

Comment:
```suggestion
                          <span class="text-highlight dir dir-ltr"
```

no need mag double underscore, wala naman syang parent block. page utilities na ganyan pwede ka magbase sa tailwind or bootstrap sa naming

Reply:
noted po